docs/tokens/nft-tokens.mdx

```diff
@@ -698,6 +703,120 @@ _For more details about how arguments have to be encoded, check [here](/develope
 
 [comment]: # (mx-context-auto)
 
+### **Modify Royalties**
+
+The manager of an ESDT token may want to set new royalities. This operation will rewrite the royalities on the specified token ID. It requires `ESDTRoleNFTModifyRoyalties` role.
+This is done by performing a transaction like this:
+
+```rust
+ModifyRoyalitiesTransaction {
+    Sender: <account address of the token manager>
+    Receiver: <same as sender>
+    Value: 0
+    GasLimit: 60000000
+    Data: "ESDTModifyRoyalties" +
+          "@" + <token identifier in hexadecimal encoding> +
+          "@" + <NFT nonce in hexadecimal encoding> +
+          "@" + <new royalties in hexadecimal encoding>
+}
+```
+
+_For more details about how arguments have to be encoded, check [here](/developers/sc-calls-format)._
+
+[comment]: # (mx-context-auto)
+
+### **Set new URIs**
+
+The manager of an ESDT token may want to set new uris. This operation will rewrite the uris on the specified token ID. It requires `ESDTRoleNFTSetNewURIs` role.
+This is done by performing a transaction like this:
+
+```rust
+SetNewURIsTransaction {
+    Sender: <account address of the token manager>
+    Receiver: <same as sender>
+    Value: 0
+    GasLimit: 60000000
+    Data: "ESDTNFTSetNewURIs" +
+          "@" + <token identifier in hexadecimal encoding> +
+          "@" + <NFT nonce in hexadecimal encoding> +
+          "@" + <list of new uris in hexadecimal encoding>
+}
+```
+
+_For more details about how arguments have to be encoded, check [here](/developers/sc-calls-format)._
+
+[comment]: # (mx-context-auto)
+
+### **Modify Creator**
+
+The creator of an NFT can be changed using a transaction like this:
+
+```rust
+ModifyCreatorTransaction {
+    Sender: <account address of the new creator that has `ESDTRoleModifyCreator` role>
+    Receiver: <same as sender>
+    Value: 0
+    GasLimit: 60000000
+    Data: "ESDTModifyCreator" +
+          "@" + <token identifier in hexadecimal encoding> +
+          "@" + <NFT nonce in hexadecimal encoding>
+}
+```
+
+_For more details about how arguments have to be encoded, check [here](/developers/sc-calls-format)._
+
+[comment]: # (mx-context-auto)
+
+### **MetaData Update**
+
+The NFT meta data can be updated using a transaction like this:
```

Comment:
Also add: `If nothing is received for a given attribute, the old version of that attribute will be kept`

Reply:
done